docs/source/models/causes/neonatal/index.rst

```diff
@@ -239,7 +282,9 @@ For simplicity, we will not include YLDs in this model.
 Validation Criteria
 +++++++++++++++++++
 
-Neonatal mortality counts and rate in simulation should match GBD estimates.
+Neonatal deaths per live birth in simulation should match same ratio derived from GBD.
+
+Relative Risk of neonatal death at specific categories of LBWSG exposure should match GBD estimates.
```

Comment:
Ideally yes, we'd like this, but we don't currently expect these to match exactly because of (1) our interpolation of the RRs, and (2) we use a constant mortality hazard at each BW,GA, whereas I think GBD doesn't assume a constant mortality hazard in each LBWSG category.